config/jobs/kubeflow/kubeflow-postsubmits.yaml

```diff
@@ -235,6 +235,8 @@ postsubmits:
         - "7200"
     annotations:
       testgrid-dashboards: sig-big-data
+      testgrid-alert-email: cloud-ml-pipeline-oncall@google.com
+      testgrid-num-failures-to-alert: "3"       
```

Comment:
@hilcj it seems common for postsubmit tests for fail for 3 times consecutively for now, shall we change this to 5?

Reply:
no problem, count changed 3 -> 5.